ui/src/Game.jsx

```diff
@@ -572,6 +620,13 @@ export class Game extends Component {
             this.leaderboardComboPressed = true;
         } else if (event.keyCode === CONSTANTS.Y_KEYCODE && event.ctrlKey) { // y + CTRL: LB on
             this.leaderboardComboPressed = false;
+        } else if (event.keyCode === CONSTANTS.D_KEYCODE && event.ctrlKey) { // d + CTRL: multiple types toggle
+            this.setState((prevState) => ({ multipleTypes: !prevState.multipleTypes }));
+        } else if (event.keyCode === CONSTANTS.S_KEYCODE && event.ctrlKey) { // s + CTRL: stream toggle
```

Comment:
Need to add a qualifier - like shift or alt. On Linux and Windows Ctrl+D and Ctrl+S mean Bookmark and Save correspondingly.